components/meta-packages/history/history

```diff
@@ -1002,6 +1002,8 @@ system/desktop/locale/zh_cn@0.5.11,5.11-2015.0.2.0
 system/desktop/locale/zh_hk@0.5.11,5.11-2015.0.2.0
 system/desktop/locale/zh_tw@0.5.11,5.11-2015.0.2.0
 system/display-manager/gdm@2.30.7,5.11-2017.0.0.9
+system/font/baekmuk-ttf@2.2-2018.0.0.1
```

Comment:
wait, we do have components/fonts/baekmuk/history

Reply:
Goot catch,
It doesn't seem to work.
Does the entry also need to have an incremented version? It now has:
system/font/baekmuk-ttf@2.2-2018.0.0.0 system/font/truetype/baekmuk

Reply:
What do you mean by 'doesn't seeem to work'? 
```
$ pkg info -r system/font/baekmuk-ttf
          Name: system/font/baekmuk-ttf
         State: Not installed (Renamed)
    Renamed to: system/font/truetype/baekmuk
     Publisher: openindiana.org
       Version: 2.2
        Branch: 2018.0.0.0
Packaging Date: November  6, 2018 at 08:17:21 AM
          Size: 0.00 B
          FMRI: pkg://openindiana.org/system/font/baekmuk-ttf@2.2-2018.0.0.0:20181106T081721Z
```

Reply:
pkg list -a|grep baekmuk
system/font/baekmuk                               2.2-2018.0.0.1             ---
system/font/baekmuk-ttf                           2.2-2018.0.0.0             i--

Reply:
Maybe it's because I have gnome-fonts installed and this requires baekmuk-ttf and freefont-ttf at the moment?

Reply:
pkg info baekmuk-ttf ?

Reply:
pkg info baekmuk-ttf
          Name: system/font/baekmuk-ttf
       Summary: Baekmuk family Korean TrueType fonts
      Category: System/Fonts
         State: Installed
     Publisher: openindiana.org
       Version: 2.2
        Branch: 2018.0.0.0
Packaging Date: Mon Feb 12 04:37:20 2018
          Size: 27.27 MB
          FMRI: pkg://openindiana.org/system/font/baekmuk-ttf@2.2-2018.0.0.0:20180212T043720Z
   Project URL: http://kldp.net/projects/baekmuk/
    Source URL: http://dlc.openindiana.org/oi-userland/source-archives/baekmuk-ttf-2.2.tar.gz

Reply:
And what if you try installing pkg://openindiana.org/system/font/baekmuk-ttf@2.2-2018.0.0.0:20181106T081721Z ?

Reply:
pkg install pkg://openindiana.org/system/font/baekmuk-ttf@2.2-2018.0.0.0:20181106T081721Z
Creating Plan (Solver einrichten): /
pkg install: No matching version of system/font/baekmuk-ttf can be installed:
  Reject:  pkg://openindiana.org/system/font/baekmuk-ttf@2.2-2018.0.0.0:20181106T081721Z
  Grund:  No version for 'require' dependency on system/font/truetype/baekmuk can be found

Reply:
Ouch, it's wrong rename package.
Font is renamed to system/font/truetype/baekmuk, but package is named system/font/baekmuk

Reply:
Does it look correct now?